libraries/SITL/SIM_Precland.cpp

```diff
@@ -98,6 +98,13 @@ const AP_Param::GroupInfo SIM_Precland::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("DIST_LMT", 8, SIM_Precland, _dist_limit, 10),
 
+    // @Param: ORIENT
+    // @DisplayName: Precland device orientation
+    // @Description: Precland device orientation vector
+    // @Values: 0:Front, 4:Back, 24:Up
```

Comment:
Should probably use our standard labels here

Reply:
Resolved as discussed on discord.